### PR TITLE
Grid component merge ready

### PR DESCRIPTION
**Pull request by b0llu**
image
image

### CODE REVIEW

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: Components/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    padding: 2rem;
+    gap: 1rem;
+    background-color: var(--card-container-bg);
+    color: var(--text-color);
+    border-radius: 5px;
+}
+
+.grid-3x3 {
```
Comment (gadgetfather) [inline Components/grid/grid.css:42]: good class names

**Review by gadgetfather — APPROVED** [review]
The code is neatly written. Great job 👍

File: Components/grid/grid.css (hunk 1 above)
Comment (b0llu) [reply Components/grid/grid.css:42]: thanks!

**Pull request merged**